leipzig-gloss.typ

```diff
@@ -98,7 +175,20 @@
         none
     }
 
-    [#gloss_number #pad(left: 1em)[#gloss_items]]
+    style(styles => {
+        block(breakable: breakable)[
+            #stack(
+            dir:ltr,
+            left_padding,
+            [#gloss_number],
+            gloss_padding - left_padding - measure([#gloss_number],styles).width,
+            [#gloss_items]
+            )
+        ]
+    }
+    )
 }
 
-#let numbered_gloss = gloss.with(numbering: true)
+#let numbered_gloss = gloss.with(numbering:true)
+
+#let nogloss = " "
```

Comment:
Is this definition doing anything?

Reply:
Oh I see your explanation. It'd be good to use this in an example, to justify why it is there.